tests/src/com.github.luben/zstd-jni/1.5.2-5/src/test/java/com_github_luben/zstd_jni/Zstd_jniTest.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+package com_github_luben.zstd_jni;
+
+import com.github.luben.zstd.Zstd;
+import com.github.luben.zstd.ZstdDictCompress;
+import com.github.luben.zstd.ZstdDictDecompress;
+import com.github.luben.zstd.ZstdDirectBufferCompressingStream;
+import com.github.luben.zstd.ZstdDirectBufferDecompressingStream;
+import com.github.luben.zstd.ZstdInputStream;
+import com.github.luben.zstd.ZstdOutputStream;
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.RandomAccessFile;
+import java.nio.ByteBuffer;
+import java.nio.channels.FileChannel;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+
+class Zstd_jniTest {
+    @Test
+    void testCompressAndDecompress() throws IOException {
+        Path path = Paths.get("src/test/resources/originTest.txt");
+        byte[] bytes = new String(Files.readAllBytes(path)).getBytes(StandardCharsets.UTF_8);
+        assertThat(bytes.length).isEqualTo(26);
+        byte[] dictBytes = new String(Files.readAllBytes(path)).getBytes(StandardCharsets.UTF_8);
+        ZstdDictCompress compressDict = new ZstdDictCompress(dictBytes, 10);
+        ZstdDictDecompress decompressDict = new ZstdDictDecompress(dictBytes);
+        byte[] compressed = Zstd.compress(bytes, compressDict);
+        assertThat(compressed.length).isEqualTo(15);
+        byte[] decompressed = Zstd.decompress(compressed, decompressDict, 1000000);
+        assertThat(decompressed.length).isEqualTo(26);
+    }
+
+    @Test
+    void testCompressFile() throws IOException {
+        File file = new File("src/test/resources/originTest.txt");
+        File outFile = new File("src/test/resources/originTest.txt.zs");
```

Comment:
It may be worth writing these files to a temporary directory so that they don't pollute the main repository

Reply:
- Do you mean let me replace it with `java.io.File.createTempFile`, and replace the tail `assertDoesNotThrow(outFile::delete);` with `assertDoesNotThrow(outFile::deleteOnExit);`? Do I need to keep committing to this PR or?

Reply:
Exactly. I would commit the change to this PR :)

Reply:
Is this done?

Reply:
> Is this done?

@gradinac Any updates?